index.html

```diff
@@ -46,30 +46,43 @@
 
     <header>
       <h1>marxist.space</h1>
-      marxist.space is a
-      <a href="https://prolesoft.github.io" target="_blank" rel="noopener nofollow noreferrer">Prolesoft</a>
-      project.
-      Check out our
-      <a href="https://prolesoft.github.io/blog" target="_blank" rel="noopener nofollow noreferrer">developer blog</a>.
-      This project is
-      <a href="https://github.com/prolesoft/marxist.space" target="_blank" rel="noopener nofollow noreferrer">free and open source</a>
-      under the
-      <a href="https://github.com/prolesoft/marxist.space/blob/master/LICENSE.md" target="_blank" rel="noopener nofollow noreferrer">LGPL-3.0</a>
-      license.
-       <a href="https://github.com/prolesoft/marxist.space/blob/master/.github/CONTRIBUTING.md" target="_blank" rel="noopener nofollow noreferrer">Contributions</a>
-       are welcome.
+
+      <div>
+        marxist.space is a <a href="https://prolesoft.github.io" target="_blank" rel="noopener nofollow noreferrer">Prolesoft</a> project.
+        <br/>
+
+        Check out our <a href="https://prolesoft.github.io/blog" target="_blank" rel="noopener nofollow noreferrer">developer blog</a>.
+        <br/>
+
+        This project is <a href="https://github.com/prolesoft/marxist.space" target="_blank" rel="noopener nofollow noreferrer">free and open source</a> under the
+        <a href="https://github.com/prolesoft/marxist.space/blob/master/LICENSE.md" target="_blank" rel="noopener nofollow noreferrer">LGPL-3.0</a> license.
+        <br/>
+        
+        <a href="https://github.com/prolesoft/marxist.space/blob/master/.github/CONTRIBUTING.md" target="_blank" rel="noopener nofollow noreferrer">Contributions</a> are welcome.
+      </div>
     </header>
 
+    <button id="scroll-top">
+      Top
+    </button>
+
+
     <main>
-      <p style="padding-left: 8px;">
-        Filter: <input id="filter" type="search" placeholder="marxism" name="search">
+      <p class="filtering">
+        <input placeholder="marxism" type="search" name="search" id="query">
+        <button id="filter">Filter</button>
       </p>
 
       <ul>
         <!-- filled from fetch -->
       </ul>
     </main>
 
+    <!-- quick scroll to top button -->
+    <script>
+      document.getElementById("scroll-top").onclick = function() {window.scrollTo(0, 0);}
```

Comment:
```suggestion
      document.getElementById("scroll-top").onclick = () => { window.scrollTo(0, 0) }
```

This project prefers ES2015+ syntax and no semicolons :)